ESP8266 build broken in current master

Just pulled current master and esp8266 build fails
```
GEN build/genhdr/qstrdefs.generated.h
FREEZE modules
build/shared-bindings/pulseio/PulseIn.o:(.text.pulseio_pulsein_obj_get_paused+0x0): undefined reference to `common_hal_pulseio_pulsein_get_paused'
build/shared-bindings/pulseio/PulseIn.o: In function `pulseio_pulsein_obj_get_paused':
PulseIn.c:(.text.pulseio_pulsein_obj_get_paused+0x18): undefined reference to `common_hal_pulseio_pulsein_get_paused'
Makefile:250: recipe for target 'build/firmware.elf' failed
make: *** [build/firmware.elf] Error 1
```

--- a/ports/esp8266/common-hal/pulseio/PulseIn.c
+++ b/ports/esp8266/common-hal/pulseio/PulseIn.c
@@ -62,6 +62,10 @@ uint16_t common_hal_pulseio_pulsein_get_maxlen(pulseio_pulsein_obj_t* self) {
     return 0;
 }
 
+bool common_hal_pulseio_pulsein_get_paused(pulseio_pulsein_obj_t* self) {
+    return true;
+}
+
 uint16_t common_hal_pulseio_pulsein_get_len(pulseio_pulsein_obj_t* self) {
     return 0;
 }

--- a/ports/esp8266/common-hal/pulseio/PulseIn.c
+++ b/ports/esp8266/common-hal/pulseio/PulseIn.c
@@ -63,7 +63,7 @@ uint16_t common_hal_pulseio_pulsein_get_maxlen(pulseio_pulsein_obj_t* self) {
 }
 
 bool common_hal_pulseio_pulsein_get_paused(pulseio_pulsein_obj_t* self) {
-    return true;
+    return false;
 }
 
 uint16_t common_hal_pulseio_pulsein_get_len(pulseio_pulsein_obj_t* self) {

